doc/introduction.rst

```diff
@@ -73,35 +65,39 @@ from `FSL official installation guide <https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Fs
     PATH=${FSLDIR}/5.0/bin:${PATH}
     export FSLDIR PATH
 
-**AFNI**: If you have access to `Neurodebian <http://neuro.debian.net>`_, simply install the `AFNI package <http://neuro.debian.net/pkgs/afni.html>`_ through Neurodebian. Otherwise, go to `AFNI installation page <https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/background_install/main_toc.html>`_. Then add the following lines of code to your  ``.bashrc`` file::
+**AFNI**: If you have access to `Neurodebian <http://neuro.debian.net>`_, then simply install the `AFNI package <http://neuro.debian.net/pkgs/afni.html>`_ through Neurodebian. Otherwise, go to `AFNI installation page <https://afni.nimh.nih.gov/pub/dist/doc/htmldoc/background_install/main_toc.html>`_. Then add the following lines of code to your  ``.bashrc`` file::
 
     # AFNI
     export PATH=/usr/lib/afni/bin:$PATH
 
-**ANTs**: The recommended way to install ANTs is to build it from source for your own system. Go to the ANTs section in `Michael Notter's excellent tutorial <http://miykael.github.io/nipype-beginner-s-guide/installation.html>`_ and follow each step. After the installation is complete, add the following lines  to your  ``.bashrc`` file ::
+**ANTs**: To install ANTS, we recommend that you build it from source for your own system. Go to the ANTs section in `Michael Notter's excellent tutorial <http://miykael.github.io/nipype-beginner-s-guide/installation.html>`_ and follow each step. When the installation is complete, add the following lines  to your  ``.bashrc`` file ::
 
     # ANTs
     export PATH=/usr/local/antsbin/bin:$PATH
     export ANTSPATH=/usr/local/antsbin/bin/
 
+**RATS**: `RATS <http://www.iibi.uiowa.edu/content/rats-overview/>`_ allows brain extraction.
 
-Installing sammba-MRI
----------------------
-For the moment, sammba-MRI has been tested only on Linux environment.
-First install the scientific Python distribution `Anaconda <https://www.anaconda.com/distribution>`_ and the version control system `Git <https://git-scm.com>`_. Then install the Python modules `nipype <https://nipype.readthedocs.io/en/latest/users/install.html>`_ and `nilearn <http://nilearn.github.io/introduction.html#installing-nilearn/>`_.
+Second: Install sammba-MRI
+===========================
 
-Sammba-mri is available as a development version. To download the source code, run the shell command::
+Sammba-MRI is available as a development version.The most recent sammba-MRI version is found on `Git <https://git-scm.com>`_ Run the following command
+as a shell command (not a Python command)::
 
     git clone https://github.com/sammba-mri/sammba-mri.git
 
-This will create a directory ``sammba-mri``. You now need to change to this directory and install the package by running (again, as a shell command)::
+Once commonly used, do not hesitate to update your copy of sammba-MRI by executing “git pull” in the samba-MRI root directory (as a shell command).
```

Comment:
No, this is maint for developers to have the most recent update. We do not want users to bother about updating and using periodically git. The way the setup install work is that it is going to install sammba on their system once for all. Then, when we package sammba for pypi there will be releases and users will update their version when they want.
Please remove this line